### ISSUE TITLE
Code clean-up

### ISSUE THREAD

**Issue by senkal** [gunnery/gunnery#45]
Hi @Eyjafjallajokull 
I know you want to do some re factoring, are you ok to do some code clean-ups first?

My propositions:
1. If you want to follow PEP 8 some changes/fixes needed.
2. Some unused import/variables.
3. General JS clean up(not necessary now, can cause potential issues but you know I like good jS :)
4. Some general fixes.

If you ok with that can prepare PR's to all of them.
It's not rocket, I just want to use some PyCharm features(code inspections report).
Please let me know what do you think.

**Comment by Eyjafjallajokull** [comment]
Hi, this would be awesome!
1. I run tool called pylint against gunnery code and it pointed to some potential fixes, and a lot of notices, It might be hard to follow, but the direction is of course correct ;)
2. Be careful when using PyCharm import cleanup feature because it broke gunnery in the past :)
3. There is not much JS, what do you want to change?

I'm really happy You want to contribute!
